src/mjolnir/speed_assigner.h

```diff
@@ -93,7 +94,7 @@ class SpeedAssigner {
         throw std::runtime_error(name + " must have " + std::to_string(entries.size()) + " speeds");
       size_t i = 0;
       for (const auto& speed : arr) {
-        entries[i++] = speed.GetUint();
+        entries[i++] = speed.IsNull() ? kUnconfiguredSpeed : speed.GetUint();
```

Comment:
this and the other several below are used to fill the speed tables with an invalid sentinal value so that when the caller asks the assigner for a speed that would have been a null it will get back that invalid value. previously calling `GetUnit` on a `null` value threw and exception and disabled the config based speed assignment entirely :frowning_face: